src/vs/workbench/contrib/debug/browser/debugSession.ts

```diff
@@ -1273,6 +1273,8 @@ export class DebugSession implements IDebugSession, IDisposable {
 						return;
 					}
 
+					this.enableDependentBreakpoints(thread);
```

Comment:
Let's move this to before the above `if (token.isCancellationRequested) {`. Even if the user resumes quickly before the DA returns the stack, we still want to enable dependent breakpoints